packages/fxa-payments-server/src/components/PaymentForm/index.tsx

```diff
@@ -353,21 +351,14 @@ const validateName: OnValidateFunction = (
 };
 
 const validateZip: OnValidateFunction = (value, focused, _props, getString) => {
-  let valid = undefined;
+  let valid = true;
   let error = null;
-  value = ('' + value).replace(/[^\d]+/g, '').substr(0, 5);
+  value = ('' + value).trim();
```

Comment:
I assume this is a string coercion. Looks funky.

Reply:
:nod: